packages/material-ui/src/AppBar/AppBar.js

```diff
@@ -85,34 +85,18 @@ const AppBar = React.forwardRef(function AppBar(props, ref) {
       )}
       ref={ref}
       {...other}
-    />
+    >
+      {children}
+    </Paper>
   );
 });
 
 AppBar.propTypes = {
-  /**
-   * The content of the component.
-   */
-  children: PropTypes.node.isRequired,
-  /**
-   * Override or extend the styles applied to the component.
-   * See [CSS API](#css) below for more details.
-   */
-  classes: PropTypes.object.isRequired,
-  /**
-   * @ignore
-   */
+  children: PropTypes.node,
+  classes: PropTypes.object,
   className: PropTypes.string,
-  /**
-   * The color of the component. It supports those theme colors that make sense for this component.
-   */
-  color: PropTypes.oneOf(['inherit', 'primary', 'secondary', 'default']),
-  /**
-   * The positioning type. The behavior of the different options is described
-   * [in the MDN web docs](https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Positioning).
-   * Note: `sticky` is not universally supported and will fall back to `static` when unavailable.
-   */
-  position: PropTypes.oneOf(['fixed', 'absolute', 'sticky', 'static', 'relative']),
+  color: PropTypes.oneOf(['default', 'inherit', 'primary', 'secondary']),
```

Comment:
Any idea on how we would handle the descriptions?

Reply:
It supports comments, add the comments to typescript and they'll be included